core/src/main/java/org/elasticsearch/action/ingest/IngestActionFilter.java

```diff
@@ -101,6 +101,7 @@ void processIndexRequest(Task task, String action, ActionListener listener, Acti
     }
 
     void processBulkIndexRequest(Task task, BulkRequest original, String action, ActionFilterChain chain, ActionListener<BulkResponse> listener) {
+        long ingestStartTime = System.currentTimeMillis();
```

Comment:
Makes sense, I can fix the bulk api to use nanoTime instead of currentTimeMillis in a separate PR.